### PR TITLE
Add AsSecureString to Get-AzAccessToken to convert the returned token to SecureString

### PR DESCRIPTION
**Pull request by msJinLei**
Fix https://github.com/Azure/azure-powershell/issues/24190

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Comment by vidai-msft** [comment]
@msJinLei Please remove the parameters EnableTestCoverage and TestCoverageLocation from all *-AzConfig help docs. They are only for internal use and are not supposed to expose to the public documents.

**Comment by msJinLei** [comment]
> @msJinLei Please remove the parameters EnableTestCoverage and TestCoverageLocation from all *-AzConfig help docs. They are only for internal use and are not supposed to expose to the public documents.

Done

File: src/Accounts/Accounts/Models/PSSecureAccessToken.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+        }
+
+        public PSSecureAccessToken(PSAccessToken accessToken)
+        {
+            Token = accessToken.Token.ConvertToSecureString();
+            ExpiresOn = accessToken.ExpiresOn;
+            TenantId = accessToken.TenantId;
+            UserId = accessToken.UserId;
+            Type = accessToken.Type;
```
Comment (vidai-msft) [inline src/Accounts/Accounts/Models/PSSecureAccessToken.cs:44]: Would this cause build error because the Type is readonly property?

**Review by vidai-msft — APPROVED** [review]
LGTM

**Pull request merged**